e2e/e2e_unbond_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (s *IntegrationTestSuite) testUnbond() {
+	s.Run("unbond", func() {
+		val1Addr, err := s.chain.validators[0].keyInfo.GetAddress()
+		s.Require().NoError(err)
+		val1 := val1Addr.String()
+
+		val2Addr, err := s.chain.validators[1].keyInfo.GetAddress()
+		s.Require().NoError(err)
+		val2 := val2Addr.String()
+
+		s.execUnbond(s.chain, 0, sdk.ValAddress(val1Addr), val1, standardFees.String(), false)
+		s.Require().Eventually(
+			func() bool {
+				batch, err := queryBatch(s.endpoint, 1)
+				s.Require().NoError(err)
+				return len(batch.BatchSignatures) == len(s.chain.validators)
```

Comment:
I did a run locally to verify my other question about the unbonding (2nd check indeed never gets called) and I noticed that the while the slices are the same length, the signature fields in the structs are empty.

Reply:
That's strange because it works on my side. 
Batch 0 has voting percentages `50000000` and `50000000`
Batch 1 has voting percentages  `49999999` and `50000000`
Batch 2 has voting percentages `50000000` and `50000000`

Reply:
It was probably a misconfiguration on my end since when we tried again the checks get called as expected. 😬